### ISSUE TITLE
Curious question that arose after reading your blog and repo.

### ISSUE THREAD

**Issue by thepenguinthatwants** [bradsease/personal-site#1]
I have used hugo nginx setup for quite long and planning to change into traefik nginx&hugo setup. As seemingly traefik cant serve static files.

I thought that ghost and hugo are like two different platforms and they use different syntax? So do you actually have two different blogs? Or have you somehow managed to mix hugo and ghost to work along?

Never used ghost so I have no clue. Also not sure if its okay to ask here about this.

**Comment by bradsease** [comment]
That's right, Hugo can be used as a blogging platform on its own, but I believe it's more general than that. It's a static site generator so it doesn't have to be just limited to blogs. I preferred the tools that Ghost provided for the blog part, so I used Hugo to only generate the main page for my site and left everything else to Ghost. The main page contains a link to the Ghost blog subdomain, but that's the only interaction between the two.

**Comment by thepenguinthatwants** [comment]
Thanks!

I understand it more now. 

There is still something bugging my mind.

      - "traefik.frontend.rule=Host:blog.example.com"

So you have nginx to serve static files so I guess its only meant for the hugo part? But then Ghost doesnt need the nginx as it can work on it own?
